domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -118,6 +118,10 @@ object InteractionObjectTestBuilder {
     return InteractionObject.newBuilder().setReal(value.toDouble()).build()
   }
 
+  fun createListOf(vararg items: String): InteractionObject {
```

Comment:
We can directly use `listOf("item a")` inside the test only. No need to do it in the `InteractionObjectTestBuilder.kt`.

Reply:
Can I use .createHtmlStringList() function from InteractionObjectTestBuilder.kt here? Because for .createListOfSetsOfHtmlStrings function, the type of arguments required is List<StringList>. If I use listOf() here, then there might be a type mismatch in the arguments for .createListOfSetsOfHtmlStrings() function.

Reply:
Sure, I guess. All the required functions are there in InteractionObjectTestBuilder file.

Reply:
Done